.ci/bl-specific.sh

```diff
@@ -4,3 +4,4 @@
 
 sudo mkdir -v -p /GPFS/CENTRAL/xf17id1/skinnerProjectsBackup/bnlpx_config/fmx_bluesky_config/
 sudo chown -Rv $USER: /GPFS/CENTRAL/xf17id1/skinnerProjectsBackup/bnlpx_config/fmx_bluesky_config/
+pip install git+https://github.com/NSLS-II-AMX/mxtools.git
```

Comment:
Ah I see. I'd suggest moving this up to the shared directory, I doubt I'll be the last one to be confused by it. No rush there though.

Reply:
That is, move mxtools into lsdc, to prevent confusion. Again, no rush.